packages/gui/src/electron/AudioThumbnailClass.ts

```diff
@@ -0,0 +1,87 @@
+import fs from 'fs';
```

Comment:
Let's remove `AudioThumbnailClass.ts` from this PR

Reply:
Done.